docs/conf/portland/2025/cfp.rst

```diff
@@ -174,13 +199,20 @@ If you'd like some guidance on how to create a talk proposal, take a look at our
 Questions?
 ----------
 
-If you have any questions, please email us at `{{email}} <mailto:{{email}}>`_ and let us know.
+If you have any questions about any part of this process, please email us at `{{email}} <mailto:{{email}}>`_, we'll be happy to chat.
 
 {% if flagcfp %}
 
 Submit your proposal
 --------------------------
 
+Please make sure you read this entire page before submitting your proposal, and pay particular attention to the following points:
+
+- **Spoilers** it's pretty normal not to want to include your main point in your abstract, but please make sure to highlight it for the selection committee!
```

Comment:
I feel like the first letter of these words should be capitalized since they're basically full sentences preceded by a bolded category.

so here, s/it's/It's  and so on

Reply:
If I have to :-p. It should for sure at least be consistent.